src/Shared/Tests/CompilerAssert.cs

```diff
@@ -91,7 +91,7 @@ public static async Task CompileAsync(string[] fileNames, string[] expectedError
 
         List<string> usingsToIgnore = new()
         {
-            "Microsoft.VisualStudio.TestTools.UnitTesting"
+            nameof(Microsoft.VisualStudio.TestTools.UnitTesting),
```

Comment:
This is not the same. This will only get you the string "UnitTesting"

Reply:
Not sure what I was thinking, thanks :)